packages/knip/src/plugins/biome/index.ts

```diff
@@ -0,0 +1,49 @@
+import type { IsPluginEnabled, Plugin, PluginOptions, ResolveConfig } from '../../types/config.js';
+import { toDeferResolve, toDependency, toIgnore, type ConfigInput, type Input } from '../../util/input.js';
+import { hasDependency } from '../../util/plugin.js';
+import type { BiomeConfig } from './types.js';
+
+const title = 'biome';
+
+const enablers = ['@biomejs/biome'];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) => 
+  {
+    const enabled = hasDependency(dependencies, enablers);
+    return enabled;
+  }
+
+const config: string[] = ['biome.json'];
+
+
+const resolveExtends = (extendsArray: string[], options: PluginOptions): Input[] => {
+  const inputs = [] as Input[];
+  for (let item of extendsArray) { 
+    // https://biomejs.dev/guides/configure-biome/#share-a-configuration-file
+    if (item.endsWith(".json")) {
+      inputs.push(toDeferResolve(item));
```

Comment:
Maybe we mean to use `toConfig()` here if they're files we also like to go through the same `resolveConfig()` function?